rfcs/models/security/confidentiality.md

```diff
@@ -25,20 +25,24 @@ As the figure below illustrates, the data sharing protocol first involves a roun
 
```

Comment:
"The client application, which is the receiver of the data, can also be privy to the view contents as it cannot tamper with the data without invalidating the accompanying proof." -> Do you want to say how the client app will use the data here rather than saying that the client app can see the data just because it can't tamper with it?

Reply:
No, there are unlimited ways in which a client app can use the data. In this context, all we are concerned about is that the client app cannot tamper with data in a way that beats its local IM's validation process. I've added a sentence to clarify.